examples/md-flexible/MDFlexParser.cpp

```diff
@@ -350,6 +400,7 @@ std::string iterableToString(T arr) {
 }
 
 void MDFlexParser::printConfig() {
+  using namespace std;
   constexpr size_t valueOffset = 32;
   cout << setw(valueOffset) << left << "Container"
```

Comment:
We could move the output operation of each name - value pair to a template function `print` which takes the name as first  and the value (type must be a template parameter) as second argument. This would make the whole section shorter, more expressive and easier to maintain.

Reply:
a new parser will be introduced anyways with the next PR, i will implement this functionality there.